docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -1087,6 +1066,7 @@ Some limitations of this analysis include:
 #. Counter to GBD, simulants can experience multiple causes of heart disease simultaneously, such as myocaridal infarction and angina. Since categories are no longer mutually exclusive, there might be an understimation of overall heart disease compared with GBD 
 #. Current documentation does not include enough information to have interventions run concurrently. This decision was made by the sim science team and Greg as it allows for multiple simplifying assumptions and removes the need for risk mediation. 
 #. To create "untreated" SBP and LDL-C values, we addded an approximate treatment value to those simulants who were initialized to be on medication. This method did not create a blanket population "PAF" from medication, which is different than other simulations. This should be checked in V&V for possible side effects.  
+#. For initialization of the model, we use the raw GBD values rather than the treatment adjusted values. Since treatment effects have not been applied yet, this is necessary and with the burn-in period it is unlikely to affect the outcomes. 
```

Comment:
To clarify, we "take measurements" from the raw gbd data instead of the adjusted values. And this is only being done for sending simulants initialized in an emergency state into the treatment ramps (which require measured values)

Reply:
Thanks, clarified!